### DISCUSSION TITLE
Download update indicator doesn't appear

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#326] [Q&A] [answered]
There is a new France map, but the map indicator (number of new maps) didn't show this). Only when manually checking for updates, then the number is shown in brackets.

**Comment by WolfgangZ42** [comment]
Here it works as designed - Even I didn't use the map recently I was informed of a new version of map and poi for France. Downloaded both - everything fine. 👍

**Comment by devemux86** [comment]
Checking for updates takes place after some seconds after the app is launched (if internet is available).
Then the indicator of the available updates appears in the menu.

@DolomitesTom 
If you have been playing with the app for a while (with internet), the update indicator appears?

**Reply by DolomitesTom** [reply]
can't say, no new update available at the moment, but have seen this already various times when checking for updates. Normally I have always internet connection when playing with settings.

**Comment by DolomitesTom** [comment]
This little issue still exists

**Reply by devemux86** [reply]
If you mean the update indicator in the menu, it checks for updates some seconds after the app starts.
It appears when we re-open the menu after the check.

![Cruiser](https://github.com/devemux86/cruiser/assets/3484020/4ec07ca3-fdcc-49f3-a6a2-c204c4c55d0e)

**Reply by DolomitesTom** [reply]
Yes this doesn't seems to work. I had 3 updates without any indication. And I don't find this download manager problem topic again. I can confirm that it happens on large map files like france and the problem is that the download is not executed in the background and if the lock screen is activated then download fails.

**Reply by devemux86** [reply]
That means:
- Wait e.g. 10" after app starts (updates are checked)
- Open or re-open the menu
- Then any updates should appear

**Reply by devemux86** [reply]
This discussion is about the update indicator?
(it's not related with the downloads size)

---

Downloads should be solved now with the new downloader (discussed in https://github.com/devemux86/cruiser/discussions/332).
Now it does not use the Android download manager's unknown process, but the full available internet bandwidth.

**Reply by DolomitesTom** [reply]
> This discussion is about the update indicator?

Yes, I have recognized that the Android download manager is not used again but this is the problem now why large map files fail now downloading. Der Download fehl your happens pretty soon after the Lockscreen is activ. I needed to download the France file 3 times and it only worked the 3rd time as I prevented the lock screen.

**Reply by devemux86** [reply]
This discussion is about the update indicator.
Is there a problem there, considering the explanation [above](https://github.com/devemux86/cruiser/discussions/326#discussioncomment-7338152) about how it works.

About the downloads, let's discuss them in their topic:
- https://github.com/devemux86/cruiser/discussions/332#discussioncomment-7338360

**Comment by DolomitesTom** [comment]
> It appears when we re-open the menu after the check.

The problem seems to be the translation of "downloads" with "herunterladen". The word is too long and then you only see three dots behind instead of the indicator.

Please change the translation into just "Laden".

**Reply by DolomitesTom** [reply]
Or put the indicator into a second Text line.

**Reply by devemux86** [reply]
This is a problem, especially if we increase the font.
Perhaps a better way is needed related to the icon.

**Comment by devemux86** [comment]
The simplest solution is to write the number of updates first:

<img src="https://github.com/devemux86/cruiser/assets/3484020/13911089-bce9-4739-a1be-750478f86e0b" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/4f791a88-73a9-459b-ad48-5217e4904fed" width="50%"/>

**Reply by devemux86** [reply, marked as answer]
> The simplest solution is to write the number of updates first

Available in [Cruiser 3.0.34](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
